Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml

```diff
@@ -570,23 +598,23 @@ script:
   - name: asm-tag-range-remove
     arguments:
     - name: range_id_list
-      description: Comma-separated list of range IDs to remove tags from.
+      description: 'Comma-separated list of range IDs to remove tags from.'
     - name: tags
       description: The name of the tags to remove from supplied IP ranges.
-    description: Removes tags from a list of IP ranges.
+    description: 'Removes tags from a list of IP ranges.'
```

Comment:
Why are you adding the single quotes here?

Reply:
Autoformat did